MatrixKit/Models/Account/MXKAccountManager.m

```diff
@@ -269,7 +269,7 @@ - (NSArray *)activeAccounts
             [activeAccounts addObject:account];
         }
     }
-    return activeAccounts;
+    return [activeAccounts copy];
```

Comment:
@morozkin Why do you copy here the new created array?

Reply:
@giomfo We have a contract that we will return NSArray and I think it’s better to follow it rather than returning NSMutableArray

Reply:
NSMutableArray inherits from NSArray, so we fulfil the contract ;). I would prefer to remove this copy...

Reply:
It is true, but it's also possible to cast returned array to mutable one and start calling mutating methods. It doesn't look like a very realistic example but there is a possibility of doing such kind of things. I will remove this copy